platformio/template_renderer.py

```diff
@@ -0,0 +1,24 @@
+import jinja2
+import json
+import sys
+
+# Read parameters
```

Comment:
In case we ever need to expand this, can we add the usual python structure of functions instead of top level code here?

I.e. can we add a `def main` and then the usual:

```python
if __name__ == "__main__":
   main()
```

This will also give us an opportunity to document the main function, we could say something about the expected command line arguments.

Reply:
Done

Reply:
Great documentation, thank you.